integ-tests/src/main/java/com/google/sql/SqlHelper.java

```diff
@@ -66,4 +68,20 @@ public static void executeQueries(Connection connection, List<String> queries)
       }
     }
   }
+
+  /**
+   * @param username DB username
+   * @param password DB password
+   * @param query A single string of a parametrized SQL request
+   */
+  public static void connectAndExecuteQueryAsUser(String username, String password, String query)
+      throws SQLException {
+    Connection connection = DriverManager.getConnection(URL_DB, username, password);
+    try (PreparedStatement preparedStatement = connection.prepareStatement(query)) {
+      preparedStatement.execute();
+    } catch (SQLException e) {
+      LOGGER.error(format("Cannot execute query: \n%s\n", query));
```

Comment:
Rather use '%n' instead of '\n' to get the system specific newline separator.

Reply:
No, it's actually all correct.
Instead of getting: 
```
Cannot execute query: CREATE SET TABLE %s.test_table_1, FALLBACK
  (Employee_Id  INTEGER)
  UNIQUE PRIMARY INDEX (Employee_Id);
```

I'd like to see:
```
Cannot execute query: 
CREATE SET TABLE %s.test_table_1, FALLBACK
  (Employee_Id  INTEGER)
  UNIQUE PRIMARY INDEX (Employee_Id);
  <a new line to separate for better visibility from other logs>
```